amida-tech/indaba-client
# Feature/INBA-41 PM All Users page

#### What's this PR do?
Implement the All Users page for project managers at `/users`

#### Related JIRA tickets:
[INBA-41](https://jira.amida-tech.com/browse/INBA-41)

#### How should this be manually tested?
1. Log in as an administrator on an instance with some users
1. Click on All Users to get to `/users`
1. Check that the users list renders with all site users
1. Check that a new user can be created with the Invite panel
1. Check that clicking on a user name shows a functioning user profile modal
1. Check that clicking on the delete icon (for a user you just added see [INBA-440](https://jira.amida-tech.com/browse/INBA-440))
1. Check that clicking on the mail icon spawns a message to the user

#### Any background context you want to provide?
This branch sits on the #218 branch, review that first

#### Screenshots (if appropriate):
